resources/scrape.py

```diff
@@ -5,7 +5,7 @@
 from database.models import ScrapedAuthor, Quote
 from utils.scraping import scrape_quotes
 from utils.utils import object_to_json
-import app_constants
+from constants import app_constants
```

Comment:
Use like - `from constants.app_constants import URL`

Reply:
I thought of doing that but I thought if we were to use multiple then the import statement will be long. 
But I will fix it as you suggested.

Reply:
Yes that's fine, but consider app_constants.py contains 100 constants. So the current import will load all 100 constants.

Reply:
Done